.travis.yml

```diff
@@ -44,30 +44,27 @@ matrix:
      language: python
      python: 3.5
      env: TESTENV=build_console
-     before_install: make deps
+     before_install:
+       - make deps
+       - pip install -r requirements_dev.txt
      script: make build_console
 
    - os: osx
      osx_image: Xcode 8.3.3
-     env: TESTENV=mac TOXENV=py27,py37
```

Comment:
This `TESTENV=mac` had so far been just an informational variable for travis. IMO it would be clearer to define at least that and probably also `TOXENV` (if the latter doesn't cause difficulties, haven't yet reviewed all) in .travis.yml instead of in Makefile, but this is just an opinion.

Reply:
fixed